reconcile/gitlab_labeler.py

```diff
@@ -45,6 +92,6 @@ def run(dry_run, gitlab_project_id=None, gitlab_merge_request_id=None):
         gl.get_merge_request_changed_paths(gitlab_merge_request_id)
     guessed_labels = guess_labels(project_labels, changed_paths)
     labels_to_add = [b for b in guessed_labels if b not in labels]
-    if labels_to_add:
+    if labels_to_add and not dry_run:
         logging.info(['add_labels', labels_to_add])
```

Comment:
Please, do log even under `dry_run` - that's the whole point of a dry run.

Reply:
I understand that you're fixing a bug in the integration. While I agree 100% with this comment, you may want to address it in a separate PR

Reply:
This will be addressed in a new pull request 👍